packages/react-ai-sdk/src/ui/use-chat/useAISDKRuntime.tsx

```diff
@@ -129,6 +182,8 @@ export const useAISDKRuntime = <UI_MESSAGE extends UIMessage = UIMessage>(
       toolInvocations.abort();
     },
     onNew: async (message) => {
+      completePendingToolCalls();
```

Comment:
AI SDK has a option called
sendAutomaticallyWhen: lastAssistantMessageHasCompleteToolCalls

with this option, the AI SDK immediately sends a request, so we end up with maybe two requests suddenly?